static/coffee/controllers.coffee

```diff
@@ -843,6 +845,6 @@ controllers.controller('PodController', ['$scope', 'PodApi', ($scope, PodApi) ->
     # TODO show failure message
 
   $scope.onTriggerSuccess = () ->
-    # TODO update notes
+    $scope.$emit('podActionSuccess')
```

Comment:
The names are a bit misleading I think. `$broadcast` sends events to down the scope hierarchy, `$emit` sends events up the scope hierarchy, so I do think we need this, though I haven't checked if simply broadcasting does work (which would mean the docs are misleading).

I do think we want this behaviour regardless though, so that we can restrict the pod controller's responsibility to pod related things.